template-manager-blueprint/src/main/resources/blueprints/configurations/metrics_monitor/shared_service.handlebars

```diff
@@ -0,0 +1,10 @@
+{
+{{{#if-true sharedService.attachedCluster}}}
+  "cluster-env": {
+    "properties": {
+      "metrics_collector_external_port": "6188",
```

Comment:
This is default port which is propagated by ambari? If yes then please remove this line

Reply:
thanks, removed the default ports